resources/js/common.js

```diff
@@ -92,7 +92,7 @@ $(function()
 
 	// Fill in examples
 	$('#example1').click(function() {
-		$('#input').val('curl canhazip.com').keyup();
+		$('#input').val('curl icanhazip.com').keyup();
```

Comment:
Why change this?

Reply:
It's the correct URL

https://major.io/2021/06/06/a-new-future-for-icanhazip/

https://github.com/major/icanhaz

Looking at `whois`, it looks like they're both on cloudflare but I don't know if they're actually run by the same person, the shortened form isn't mentioned on any blog posts.

Reply:
I can't remember how I found it or why I used it, but I think someone recommended it to me... if I recall it was an obscure reason...

Sorry, been busy; will try to look at this when I can.